### ISSUE TITLE
General Cleanup / Improvements

### ISSUE THREAD

**Issue by sodoku** [c-base/c-beam#24]
This is a WIP list of things that could be improved or need cleanup. I will try to updated this regularly.

**Comment by sodoku** [comment]
# mqtt websocket port not documented

Our mqtt server also supports connections via websocket, but it doesn't seem to be documented anywhere.

Possible Solutions:

- Update https://wiki.c-base.org/dokuwiki/projects:mqtt to include the websocket port

**Comment by sodoku** [comment]
# bar views are outdated (and broken)

The views for bar prices and calculations don't seem to work anymore. They have also already been replaced by the current cash system and their apis/views.

```python
url(r'^bar/preise$', views.bar_preise, name='bar_preise'),
url(r'^bar/leergut$', views.bar_leergut, name='bar_leergut'),
url(r'^bar/calc$', views.bar_calc, name='bar_Calc'),
url(r'^bar/abrechnung$', views.bar_abrechnung, name='bar_abrechnung'),
```

Possible Solutions:

- Remove the views/routes

**Comment by sodoku** [comment]
# mqtt doesn't seem to support user list

It looks like you can only get boarding information via mqtt but no way to get the current user list.

Possible Solutions:

- Add mqtt topic, e.g. `user/list`

**Comment by saces** [comment]
TL;DR: This thing is very outdated; its needs a revamp from scratch..  lets chat in matrix about it?

**Comment by cmile** [comment]
> TL;DR: The telemetry server is a bit outdated; its needs a revamp from scratch.. lets chat in matrix about it?

I disagree with "revamping from scratch", I have already a working version with up-to-date dependencies, just the host needs to be updated before. Also I have started to add a REST API using django-rest-framework. You can find it in the branch named django4.

**Comment by sodoku** [comment]
I decided to create individual issues instead as it's easier to track them that way.
